processor/deltatocumulativeprocessor/internal/data/expo/scale_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package expo_test
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/matryer/is"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor/internal/data/expo"
+)
+
+func TestDownscale(t *testing.T) {
+	type Repr[T any] struct {
+		scale expo.Scale
+		bkt   T
+	}
+
+	cases := [][]Repr[string]{{
+		{scale: 2, bkt: "1 1 1 1 1 1 1 1 1 1 1 1"},
+		{scale: 1, bkt: " 2   2   2   2   2   2 "},
+		{scale: 0, bkt: "   4       4       4   "},
+	}, {
+		{scale: 2, bkt: "ø 1 1 1 1 1 1 1 1 1 1 1"},
+		{scale: 1, bkt: " 1   2   2   2   2   2 "},
+		{scale: 0, bkt: "   3       4       4   "},
+	}, {
+		{scale: 2, bkt: "ø ø 1 1 1 1 1 1 1 1 1 1"},
+		{scale: 1, bkt: " ø   2   2   2   2   2 "},
+		{scale: 0, bkt: "   2       4       4   "},
+	}, {
+		{scale: 2, bkt: "ø ø ø ø 1 1 1 1 1 1 1 1"},
+		{scale: 1, bkt: " ø   ø   2   2   2   2 "},
+		{scale: 0, bkt: "   ø       4       4   "},
+	}, {
+		{scale: 2, bkt: "1 1 1 1 1 1 1 1 1      "},
+		{scale: 1, bkt: " 2   2   2   2   1     "},
+		{scale: 0, bkt: "   4       4       1   "},
+	}, {
+		{scale: 2, bkt: "1 1 1 1 1 1 1 1 1 1 1 1"},
+		{scale: 0, bkt: "   4       4       4   "},
+	}}
+
+	type B = expo.Buckets
+	for i, reprs := range cases {
+		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
```

Comment:
minor. You can add `t.Parallel()` to your tests to allow them to run in parallel

Reply:
those tests complete in milliseconds, so I felt like it was not worth ti to make them parallel because of the subtle data races that _may_ happen if one is not careful

Reply:
Fair